src/components/TopBar/TopBarStyles.js

```diff
@@ -0,0 +1,71 @@
+import styled from 'styled-components';
+
+export const StyledTopBarWrapper = styled.nav`
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 100%;
+  height: 57px;
+  z-index: 2;
+`;
+
+export const StyledTopBar = styled.div`
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 100%;
+  height: 56px;
+  background-color: #2a2a2a;
+  box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.12), 0px 2px 4px rgba(0, 0, 0, 0.14),
+    0px 1px 2px rgba(0, 0, 0, 0.2);
+  z-index: 3;
+`;
+
+export const StyledHeader = styled.header`
+  position: absolute;
+  width: 176px;
+  height: 24px;
+  left: 69px;
+  top: 16px;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 19px;
+  line-height: 22px;
+  display: flex;
+  align-items: center;
+  color: #3c78c8;
+`;
+
+export const StyledControls = styled.div`
+  position: absolute;
+  width: 99px;
+  height: 24px;
+  right: 16px;
+  top: 16px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+export const StyledIcon = styled.img`
+  position: relative;
+  width: ${props => props.width || '24px'};
+  height: ${props => props.height || '24px'};
+`;
+
+export const StyledGradientLine = styled.div`
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  width: 100%;
+  height: 1px;
```

Comment:
If we're adding a `1px` line with `position: absolute` at the bottom of an element, should we also set `margin-bottom: 1px` ? To ensure that the element and the bottom doesn't get cut by `1px` by this line.

Reply:
I took into account the height of this line, when set StyledTopBarWrapper height.

StyledTopBarWrapper's height = StyledTopBar's height + StyledGradientLine's height
57px=56px+1px

Reply:
Oh... Well that's not really obvious :)
Ideally we'd have a variable like  `bottomLineHeight`.
Or at least a comment
```
// 57px = 56px top bar height + 1px bottom gradient line height
height: 57px;
```
With variables would look nicer :)
```
height: topBarHeight + topBarBottomLineHeight
```

Reply:
It would be great to create file with variables, that are used more than once. But do you think, that 'topBarBottomLineHeight' must be there too?

Reply:
Without `topBarBottomLineHeight` variable we will have `height: topBarHeight + 1`.
Looking at this one might think

![image](https://user-images.githubusercontent.com/6686679/83056304-d38f2d80-a05d-11ea-85c9-53819d57e6eb.png)

We don't have to create a separate file for variables.
We need it only here for now. So just declare it in this file
```
// height of the gradient line at the very bottom of the TopBar
const bottomLineHeight = 1;

const StyledGradientLine = styled.div`
  height: ${bottomLineHeight}px;
`

Reply:
Ok) Sure, I'll do it)

Reply:
So, do I need to create "topBarHeight" constant too?